src/frontend/Komponenter/Behandling/Simulering/SimuleringOversikt.tsx

```diff
@@ -29,7 +29,7 @@ export const RadVerdi = styled.td`
 `;
 
 export const ResultatVerdi = styled(BodyShortSmall)<{ verdi: number }>`
-    color: ${(props) => (props.verdi > 0 ? navFarger.navGronn : navFarger.redError)};
+    color: ${(props) => (props.verdi > 0 ? NavdsGlobalColorGreen500 : NavdsGlobalColorRed500)};
```

Comment:
Det nye designsystemet har egne farger for tekst eks: `var(--a-text-danger)` som kanskje kunne blitt brukt her?
https://aksel.nav.no/designsystem/side/tokens-semantiske-farger#hae8701b4dd57

Reply:
Dessverre finnes ikke disse i v.1.5 som vi bruker, men v.2.0 - og oppgradering er jeg usikker på om funker så bra før felles-komponentene har blitt bumpet 😬